src/components/TodoList.tsx

```diff
@@ -0,0 +1,83 @@
+import classNames from 'classnames';
+import React from 'react';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  todos: Todo[]
+  deleteTodo: (id:number) => void
+  tempTodo: Todo | null
+};
+
+export const TodoList: React.FC <Props> = ({
+  todos,
+  deleteTodo,
+  tempTodo,
+}) => {
+  return (
+    <section className="todoapp__main" data-cy="TodoList">
+      {todos.map(todo => (
+        <div
+          key={todo.id}
+          data-cy="Todo"
+          className={classNames('todo', { completed: todo.completed })}
+        >
+          <label className="todo__status-label">
+            <input
+              onClick={() => {}}
+              data-cy="TodoStatus"
+              type="checkbox"
+              className="todo__status"
+              checked={todo.completed}
+            />
+          </label>
+
+          <span data-cy="TodoTitle" className="todo__title">
+            {todo.title}
+          </span>
+          <button
+            onClick={() => {
+              deleteTodo(todo.id);
+            }}
+            type="button"
+            className="todo__remove"
+            data-cy="TodoDelete"
+          >
+            ×
+          </button>
+
+          <div data-cy="TodoLoader" className="modal overlay">
+            <div className="modal-background has-background-white-ter" />
+            <div className="loader" />
+          </div>
+        </div>
```

Comment:
Also the loader should be each time you add or delete Todos (and in other cases)

Check the working version

![image](https://github.com/mate-academy/react_todo-app-add-and-delete/assets/91946036/6e38329c-fa47-4ca8-9544-c84a6d7bccd7)